src/frontends/common/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ else()
     set(FRONTEND_LIB_SUFFIX "${FRONTEND_LIB_SUFFIX}${CMAKE_SHARED_LIBRARY_SUFFIX}${OpenVINO_VERSION_SUFFIX}")
 endif()
 
-target_compile_definitions(${TARGET_NAME}_obj PRIVATE
+target_compile_definitions(${TARGET_NAME}_obj PUBLIC
```

Comment:
@ilya-lavrenov @ilyachur please, take a look. I need `PREFIX` and `SUFFIX` to determine FE file name inside tests, maybe you have any ideas how to do it better?

Reply:
Why cannot you add these prefix and suffix to test binary?

Reply:
Done. But I deliberately ignore `OpenVINO_VERSION_SUFFIX` since there is also a symlink pointed to versioned library